sources/xxtea/xxtea.h

```diff
@@ -20,6 +20,11 @@
 #define XXTEA_H
 
 #include <stddef.h> /* for size_t & NULL declarations */
+#include "platform/CCPlatformMacros.h"
```

Comment:
第三方依赖引擎的头文件真的好么？
我建议把CC_DLL的实现copy过来这里，定义为XXTEA_DLL

Reply:
我是参考 tinyxml 的实现。。

https://github.com/cocos-creator/cocos2d-x-lite-external/blob/master/sources/tinyxml2/tinyxml2.h#L27

Reply:
So the question is coming, is it correct to continue to write the worse code, or correct it in the new module?

Reply:
Should we define XX_DLL for every third lib?

Reply:
Yep, it's better. We should not modify source code of third-party libraries, it will be easier for us to upgrade third party libraries. If need to modify indeed, it's better to send a Pull Request to official.
For now, I think just don't bring more these hack code is ok.

Reply:
One rule: Each module should be independent with other modules.
We didn't follow this strictly before, but let's make it better in the future.